pymathics/graph/eval/parametric.py

```diff
@@ -36,7 +36,7 @@ def eval_full_rary_tree(
     self, r: Integer, n: Integer, expression, evaluation: Evaluation, options: dict
 ) -> Optional[Graph]:
     """
-    Call networkx to get a full_raray_tree using parameters, ``r`` and ``t``.
+    Call networkx to get a full_rary_tree using parameters, ``r`` and ``t``.
```

Comment:
"full r-ary tree" would be easier to understand. 'rary' sound to me like "weirdy"

Reply:
networkx `full_rary_tree` is NetworkX's name. However I will revise this. I agree that something like full r-ary tree would be be better.

Reply:
Change that - lookin at this this is  pretty thin wrapper around  NetworkX's `full_rary_tree`. So I think best here to just make it clear that this is what it is doing. And you you want more detail about `full_rary_tree`  looking at the docs in networkx would be better.

Reply:
Thanks. And OK, but in any case, there is a link to the networkx documentation.

Reply:
I think we are on the same page, but just to make sure...

If a number of IDE's and editors that support LSP (language server protocol), when you hover over "full_rary_tree"  in the code it will take you to the definition  or show you the docstring.